client/lib/analytics/test/gpc-opt-out.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+import { mayWeTrackUserGpcInCCPARegion, isGpcFlagSetOptOut } from '../utils';
+import isRegionInCcpaZone from '../utils/is-region-in-ccpa-zone';
+
+// Return a predictable value for whether the user is in a CCPA region.
+jest.mock( '../utils/is-region-in-ccpa-zone' );
+
+let windowSpy = jest.SpyInstance;
+
+// Return a predictable value for window.navigator.globalPrivacyControl.
+const setMockGPCValue = ( gpcValue ) => {
```

Comment:
We should either write "GPC" in function names uppercase or only capitalized. I vote for the latter, as we do with `isRegionInCcpaZone`.